deploy-manage/deploy/elastic-cloud/cloud-hosted.md

```diff
@@ -50,10 +51,159 @@ $$$faq-where$$$
-* [/raw-migrated-files/cloud/cloud/ec-prepare-production.md](/raw-migrated-files/cloud/cloud/ec-prepare-production.md)
-* [/raw-migrated-files/cloud/cloud/ec-faq-getting-started.md](/raw-migrated-files/cloud/cloud/ec-faq-getting-started.md)
-* [/raw-migrated-files/cloud/cloud/ec-about.md](/raw-migrated-files/cloud/cloud/ec-about.md)
-* [/raw-migrated-files/cloud/cloud-heroku/ech-configure.md](/raw-migrated-files/cloud/cloud-heroku/ech-configure.md)
\ No newline at end of file
+It is also formerly known as Elasticsearch Service.
+
+{{ech}} allows you to manage one or more instances of the Elastic Stack through **deployments**. These deployments are hosted on {{ecloud}}, through the cloud provider and regions of your choice, and are tied to your organization account.
+
+A **hosted deployment** helps you manage an Elasticsearch cluster and instances of other Elastic products, like Kibana or APM instances, in one place. Spin up, scale, upgrade, and delete your Elastic Stack products without having to manage each one separately. In a deployment, everything works together.
+
+::::{note}
+{{ech}} is one of the two deployment options available on {{ecloud}}. [Depending on your needs](../elastic-cloud.md), you can also run [Elastic Cloud Serverless projects](/deploy-manage/deploy/elastic-cloud/serverless.md).
+::::
+
+
+**Hardware profiles to optimize deployments for your usage.**
+
+You can optimize the configuration and performance of a deployment by selecting a **hardware profile** that matches your usage.
+
+*Hardware profiles* are presets that provide a unique blend of storage, memory and vCPU for each component of a deployment. They support a specific purpose, such as a hot-warm architecture that helps you manage your data storage retention.
+
+You can use these presets, or start from them to get the unique configuration you need. They can vary slightly from one cloud provider or region to another to align with the available virtual hardware.
+
+**Solutions to help you make the most out of your data in each deployment.**
+
+Building a rich search experience, gaining actionable insight into your environment, or protecting your systems and endpoints? You can implement each of these major use cases, and more, with the solutions that are pre-built in each Elastic deployment.
+
+:::{image} ../../../images/cloud-ec-stack-components.png
+:alt: Elastic Stack components and solutions with Enterprise Search
+:width: 75%
+:::
+
+:::{important}
+Enterprise Search is not available in {{stack}} 9.0+.
+:::
+
+These solutions help you accomplish your use cases: Ingest data into the deployment and set up specific capabilities of the Elastic Stack.
+
+Of course, you can choose to follow your own path and use Elastic components available in your deployment to ingest, visualize, and analyze your data independently from solutions.
+
+
+## How to operate {{ech}}? [ec_how_to_operate_elasticsearch_service]
```

Comment:
we don't have to do this now, but this section should be refreshed to make it clear what the scope of the section is vs. what now lives elsewhere.

Reply:
Not exactly sure if that's necessary given that we have docs a bit everywhere now. As long as the recipe makes sense. I simplified it a bit to remove some redundant parts or items that felt out of place.